app/src/androidTest/java/org/mozilla/fenix/ui/SettingsBasicsTest.kt

```diff
@@ -86,45 +92,63 @@ class SettingsBasicsTest {
         }
     }
 
-    @Ignore("This is a stub test, ignore for now")
     @Test
     fun selectNewDefaultSearchEngine() {
-        // Open 3dot (main) menu
-        // Select settings
-        // Select "Search engine"
-        // Choose: DuckDuckGo
-        // Back arrow to Home
-        // Verify DuckDuckGo icon in Navigation bar
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openSearchSubMenu {
+            enableDuckDuckGoAsSearchEngine()
+        }.goBack {
+        }.goBack {
+            verifyDefaultSearchEngineIsDDG()
+        }
     }
 
-    @Ignore("This is a stub test, ignore for now")
     @Test
     fun toggleSearchSuggestions() {
-        // Enter: "mozilla" in navigation bar
-        // Verify more than one suggesion provided
-        // Open 3dot (main) menu
-        // Select settings
-        // Select "Search engine"
-        // Toggle 'Show search suggestions' to 'off'
-        // Back arrow twice to home screen
-        // Enter: "mozilla" in navigation bar
-        // Verify no suggestions provided
+        homeScreen {
+        }.openNavigationToolbar {
+            verifySuggestionsAreMoreThan(1, "mozilla")
```

Comment:
I think we should include "search" in the function name, `verifySearchSuggestions...`